chore: add note about restoring management cluster backup (#4183)
mesosphere/dcos-docs-site: pages/dkp/kommander/2.1/backup-and-restore/index.md

velero restore create --from-backup BACKUP-NAME
```

> #### Restoring Kommander Management Cluster

Reviewer: ```suggestion
#### Restore Kommander Management Cluster
```

> <p class="message--note"><strong>NOTE: </strong>When restoring a backup to a Kommander Management Cluster that is running Kommander installation it is necessary to adjust configuration to avoid restore errors.

Reviewer 2: ```suggestion
<p class="message--note"><strong>NOTE: </strong>When restoring a backup to the Kommander Management Cluster, you must adjust configuration to avoid restore errors.
```

Or do we call the management cluster “Host Cluster” in docs? (I think this is inconsistent across docs, maybe we should pick one and change it everywhere)

Author: The modified page is using the phrase "Kommander Management Cluster".

</p>

> Before running a restore make sure that the following `ResourceQuota` configuration is not configured on your cluster:

Reviewer: ```suggestion
Before restoring a backup, ensure the following `ResourceQuota` configuration is not configured on your cluster:
```

> ```sh

Reviewer: We are using bash for input and sh for output, so the copy icon only appears in executable commands and customers don't try to execute output :) 
```suggestion
```bash
```

kubectl -n kommander delete resourcequota one-kommandercluster-per-kommander-workspace
```

> When the restore is completed the manually removed `ResorceQuota` named `one-kommandercluster-per-kommander-workspace` will be automatically restored.

Reviewer 2: ```suggestion
When the restore is completed, the manually removed `ResourceQuota` named `one-kommandercluster-per-kommander-workspace` will automatically be restored.
```

> You must also turn off the `Workspace` validation webhooks which would prevent restoring Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present it must be modified with the command:

Reviewer: Since I don't know much in the topic, I see two different interpretations, so here are my suggestions: 
```suggestion
Turn off the `Workspace` validation webhooks. Otherwise, you will not restore Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present, it must be modified with the command:
```
```suggestion
Turn off the `Workspace` validation webhooks to prevent restoring Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present, it must be modified with the command:
```

> ```sh

Reviewer: ```suggestion
```bash
```

> It is also necessary to turn off the `Workspace` validation webhooks which would prevent restoring Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present it must be modified with the command:

Reviewer 2: ```suggestion
You must also turn off the `Workspace` validation webhooks which would prevent restoring Workspaces with pre-configured namespaces. If the validation webhook named `kommander-validating` is present it must be modified with the command:
```

kubectl patch validatingwebhookconfigurations kommander-validating \
  --type json \
  --patch '[
    {
      "op": "remove",
      "path": "/webhooks/0/rules/3/operations/0"
    }
  ]'
```

> When the restore is completed, you must re-add the removed `CREATE` webhook rule operation with:

Reviewer: ```suggestion
When the backup is completed, re-add the removed `CREATE` webhook rule operation with:
```

> ```sh

Reviewer: ```suggestion
```bash
```

> When the restore is completed it is necessary to re-add the removed `CREATE` webhook rule operation with:

Reviewer 2: ```suggestion
When the restore is completed, you must re-add the removed `CREATE` webhook rule operation with:
```
